### DISCUSSION TITLE
Highlight Active Relationship Line

### DISCUSSION THREAD

**Discussion by Chinchivi** [vslinko/obsidian-outliner#173] [Ideas]
Hi there. Maybe this is somethig for the outliner plugin: [Highlight Active Relationship Line](https://forum.obsidian.md/t/help-needed-highlight-active-relationship-line/20957)?

It seems that the wished functionality can be implemented via css in Obsidian. But how about the have it directly in the plugin? It basically should do the two following thins:

1. To highlight the current line I’m writing on.
2. When outlining: To also highlight the currently active relationship line as seen in the screenshots.

Best!
Chichivi

**Comment by vslinko** [comment]
@Chinchivi Hi, thanks for the idea!
